test/plugins/features/movies/controller.test.js

```diff
@@ -23,14 +43,67 @@ describe('movie controller', () => {
 
   });
 
-  describe('get', () => {
+  describe('list', () => {
+
+    it('list movies released after year_start', () => {
+      const payload = { year_start: 1983 };
+
+      return Controller.list(payload)
+      .then((json) => {
+        json.map((movie) => {
+          expect(movie.attributes.release_year).to.be.at.least(1983);
+        });
+
+        return json;
```

Comment:
Since this is a test and this is the last callback of the promise chain, it's resolved value will just be discarded, so you don't need to return this at the end.